services/helper.py

```diff
@@ -8,9 +7,20 @@ def __init__(self, repository: HelperRepository, logger: logging.Logger):
         self.logger = logger
         self.repository = repository
 
-    def handle_thankyou(self, message: discord.Message):
-        thanked_users: list[discord.Member] = message.mentions
-        for user in thanked_users:
-            self.repository.add_points_to_user(user.id, 1)
-            self.logger.info(f'User {user.display_name} got {1} point from '
-                             f'{message.author.display_name} for being helpful')
+    def add_points(self, thanked, thanker, bot, p):
+        if thanked != thanker and thanked != bot:
```

Comment:
`if thanked not in {thanker, bot}` pierdoła, ale czemu nie optymalizować nawet pierdół